### DISCUSSION TITLE
Chat : why SSL optional and disabled by default ?

### DISCUSSION THREAD

**Discussion by KaKi87** [twireapp/Twire#386] [Q&A] [answered]
Hello,

I just found out that there's an unchecked "use SSL for the connection" option in chat settings.

Why is it disabled by default, and why can it be disabled at all ?

Isn't that basic security ?

Unless of course, it's literal, and means downgrading from TLSv1.2 to SSLv3, which, of course, shouldn't be enabled by default.

Thanks

**Comment by samfundev** [comment, marked as answer]
I'm not sure, but if I had to guess I would say it's because of older devices that don't support newer versions of SSL. But I do think it should be enabled by default since I suspect most devices would support it. I've enabled it by default in this commit: 60a66f1f5cc6db2a4aee63b75c85db737998c9dd.
